include/zephyr/net/wifi_mgmt.h

```diff
@@ -184,6 +190,12 @@ struct wifi_ps_mode_params {
 	enum wifi_ps_mode mode;
 };
 
+#define WIFI_MIN_PS_TIMEOUT 0
+#define WIFI_MAX_PS_TIMEOUT 5
+struct wifi_ps_timeout_params {
+	int timeout_ms;
```

Comment:
Why not use the `k_timeout_t` type?

Reply:
The firmware expects the parameter as integer.